lib/tasks/migrate_pacbio_aliquot_data.rake

```diff
@@ -126,6 +126,50 @@ namespace :pacbio_aliquot_data do
     end
   end
 
+  task migrate_well_data: :environment do
```

Comment:
Do we want to add a primary aliquot for wells? I havent added one because I am not sure on the value it adds a well is not something that can be aliquoted itself.

Reply:
No. I don't think it makes sense either.